MemeMaker.html

```diff
@@ -74,6 +74,22 @@
             function saveFile() {
                 window.open( document.querySelector( 'canvas' ).toDataURL() );
             }
+            
+            function isValidFile(data)
+            {
+            	var mimeType = getMimeType(data);
+            	var allowedMimeTypes = ["image/jpeg", "image/jpg", "image/png", "image/gif"];
```

Comment:
Bitmap is not a standard format for the web, do you have a bmp image? May be can give a try by analyzing it's data uri